arbitrary-users-patch/build_images.sh

```diff
@@ -31,9 +31,9 @@ while read -r line; do
   base_image_name=$(echo "$line" | tr -s ' ' | cut -f 1 -d ' ')
   base_image=$(echo "$line" | tr -s ' ' | cut -f 2 -d ' ')
   echo "Building ${NAME_FORMAT}/${base_image_name}:${TAG} based on $base_image ..."
-  docker build -t "${NAME_FORMAT}/${base_image_name}:${TAG}" --no-cache --build-arg FROM_IMAGE="$base_image" "${SCRIPT_DIR}"/
+  docker build -t "${NAME_FORMAT}/${base_image_name}:${TAG}" --no-cache --build-arg FROM_IMAGE="$base_image" "${SCRIPT_DIR}"/ | cat
```

Comment:
is it related to `--progress=plain` ?

Reply:
BTW looks like you could add an alias to docker executable as well https://github.com/moby/moby/issues/3694#issuecomment-363884716 (so no need to append cat on every instructions)

Reply:
It looks like `--progress=plain` would also do what we need, though AFAICT it's an "experimental" option, and not available in older versions.

I'd prefer not to redirect *all* output through `cat`, since that's less clear IMO. Most tools can figure out they're not in a tty.

Reply:
you can maybe just alias `docker` command